src/api/db/structure.sql

```diff
@@ -1359,6 +1361,7 @@ INSERT INTO schema_migrations (version) VALUES
 ('20141201135426'),
 ('20141202135426'),
 ('20141208135426'),
+('20141302101042'),
```

Comment:
This migration does not exist. @adrianschroeter do you have any idea why this was added here? Perhaps the migration only exists on production?